tf-modules/archive/publish_reports.tf

```diff
@@ -114,7 +113,6 @@ resource "aws_lambda_function" "publish_reports" {
   environment {
     variables = {
       ExecutionsTable         = var.dynamo_tables.executions.name
```

Comment:
It also doesn't need this environment variable anymore, I don't believe

Reply:
Fixed